src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -1448,9 +1483,20 @@ private static bool WriteScriptBlockToLog(ScriptBlock scriptBlock, int segment,
                             // attacker seeing potentially sensitive data. Because if they aren't detected, then
                             // they can just wait on the compromised box and see the sensitive data eventually anyways.
 
-                            string errorMessage = StringUtil.Format(SecuritySupportStrings.CouldNotEncryptContent, textToLog, error.ToString());
-                            PSEtwLog.LogOperationalError(PSEventId.ScriptBlock_Compile_Detail, PSOpcode.Create, PSTask.ExecuteCommand, PSKeyword.UseAlwaysOperational,
-                                            0, 0, errorMessage, scriptBlock.Id.ToString(), scriptBlock.File ?? string.Empty);
+                            string errorMessage = StringUtil.Format(
+                                SecuritySupportStrings.CouldNotEncryptContent,
+                                textToLog,
+                                error.ToString());
+                            PSEtwLog.LogOperationalError(
+                                id: PSEventId.ScriptBlock_Compile_Detail,
+                                opcode: PSOpcode.Create,
+                                task: PSTask.ExecuteCommand,
+                                keyword: PSKeyword.UseAlwaysOperational,
+                                0,
+                                0,
```

Comment:
Please use named parameters.

Reply:
Those last two are `params args`; not sure if applying names to a `params` argument actually works? I'll see if omnisharp complains.

Reply:
Ok, ignore my request.